### DISCUSSION TITLE
q-select and label-color

### DISCUSSION THREAD

**Discussion by Jejec** [quasarframework/quasar#13559] [General - Components / Directives / etc] [answered]
Hello,

I put a q-select component in my navbar but the text is displayed in black instead of white like the rest. I figured okay I'll use the label-color="secondary" option to change the color to something lighter, but no change is showing.
I may have misunderstood the option but I don't see how then to do what I want? Or is it a bug?

Thanks for your help

**Comment by pdanpdan** [comment]
Use prop `dark`

**Reply by Jejec** [reply]
Hi,

Unfortunately not, because it also changes the background colors of the select at the options to black, which I don't want.

**Comment by pdanpdan** [comment]
Make a codepen

**Comment by Jejec** [comment]
I'm not familiar with codepen but it's easy to reproduce the problem.

You need a select with:
`      <q-select
        v-model="data"
        :options="options"
        dense
        label-color="negative"
        emit-value
        map-options
      />`

And we can see that the select option is not red like ask with label-color.
Where did I misunderstand the label-color option ?

**Comment by Jejec** [comment]
Actually, I had misunderstood the option. I tried your solution, I was able to change the color. Thank you very much for your time.

**Comment by pratik227** [comment]
`<q-select
        filled
        v-model="single"
        :options="options"
        label="Single" class="bg-orange"
        style="width: 250px" dark text-color="white"
                :options-dark="false"
      ></q-select>`

https://codepen.io/Pratik__007/pen/mdXjQMm

**Comment by metalsadman** [comment, marked as answer]
label-color is for the label, if you want the selected text, can do it in css 
```css
.q-field__native span
  color: $negative
```
or slot in selected slot. if you are styling for the dropdown options, then use props `popup-content-class` or `popup-content-style`.
